<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanTrans</class>
 <widget class="QMainWindow" name="LanTrans">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LanTrans</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>45</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="receiveFileLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="sendFileRadio">
       <property name="text">
        <string>Send Files</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addFileBtn">
       <property name="text">
        <string>Add File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBtn">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="receiveFileRadio">
       <property name="text">
        <string>Receive Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePathBtn">
       <property name="text">
        <string>SavePath</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="statusLayout">
     <item>
      <widget class="QLabel" name="status">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="statusText"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>209</y>
      <width>471</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="processdureLayout">
     <item>
      <widget class="QLabel" name="processdure">
       <property name="text">
        <string>Processdure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="fileList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>151</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="actionLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="connectHostBtn">
       <property name="text">
        <string>Connect Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
